.github/workflows/check_sri_integrity_webassets.yml

```diff
@@ -0,0 +1,51 @@
+name: Check SRI integrity for webassets.iota.org
+
+on:
+  schedule:
+    - cron: "0 22 * * *" # daily at 10:00 PM (UTC)
```

Comment:
I would call it from turbo_nightly.yml 

Make sure to add here the `workflow_call` thingy, you can check _e2e.yml on how to do it